indy_node/server/node.py

```diff
@@ -116,6 +139,119 @@ def __init__(self,
         self.actionReqHandler.poolCfg = self.poolCfg
         self.actionReqHandler.restarter = self.restarter
 
+    def init_idr_cache_storage(self):
```

Comment:
I think it would be great to move all request-handlers + storages-specific logic into a separate class/file/util